### PR TITLE
Conditional references!

### PR DESCRIPTION
**Pull request by Johnny-Bee**
Conditional references!

### CODE REVIEW

**Comment by tjaskula** [comment]
Related to https://github.com/vlingo-net/vlingo-net-platform/issues/1

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <VlingoVersion>0.6.4.6</VlingoVersion>
+  </PropertyGroup>
+</Project>
```
Comment (tjaskula) [inline Directory.Build.props]: @Johnny-Bee Just for me to understand. If I'm only working on `vlingo-net-uuid` repo. How the version from this file will be injected to `Vlingo.UUID.Tests.csproj` ? I cannot see it is included

File: src/Vlingo.UUID.Tests/Vlingo.UUID.Tests.csproj [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <LangVersion>latest</LangVersion>
+    <Configurations>Debug;Release;Debug With Project References</Configurations>
```
Comment (tjaskula) [inline src/Vlingo.UUID.Tests/Vlingo.UUID.Tests.csproj]: This is only used by the platform right ?

Comment (Johnny-Bee) [reply src/Vlingo.UUID.Tests/Vlingo.UUID.Tests.csproj]: Yes

Comment (Johnny-Bee) [reply src/Vlingo.UUID.Tests/Vlingo.UUID.Tests.csproj]: Yes

File: Directory.Build.props (hunk 1 above)
Comment (Johnny-Bee) [reply Directory.Build.props]: The test projects are using project reference so no worries

Comment (Johnny-Bee) [reply Directory.Build.props]: The repos are still independent

Comment (Johnny-Bee) [reply Directory.Build.props]: Yes the usage of project references is triggered when you use the solution configuration Debug with Project References

Comment (tjaskula) [reply Directory.Build.props]: Seems like this is imported directly during the build according to https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019

**Pull request merged**